spec/system/support/prior_authority/step_helpers.rb

```diff
@@ -167,6 +195,42 @@ def fill_in_service_cost(cost_type: :variable)
 
       click_on 'Save and continue'
     end
+
+    def fill_in_alternative_quote_question
+      choose 'Yes'
+
+      click_on 'Save and continue'
+    end
+
+    def fill_in_alternative_quote
+      fill_in 'Contact full name', with: 'Jim Bob'
+      fill_in 'Organisation', with: 'Experts Inc.'
+      fill_in 'Postcode', with: 'SW1A 1AA'
+
+      choose 'Charged per item'
+      fill_in 'Number of items', with: '1'
+      fill_in 'What is the cost per item?', with: '100'
+      fill_in 'prior_authority_steps_alternative_quotes_detail_form_travel_time_1i', with: '1'
+      fill_in 'prior_authority_steps_alternative_quotes_detail_form_travel_time_2i', with: '0'
+      fill_in 'What is the hourly cost?', with: '50'
+      fill_in 'Total additional costs', with: '5'
+
+      click_on 'Save and continue'
+
+      within('.govuk-form-group', text: 'Do you want to add an additional quote?') do
+        choose 'No'
+      end
+      click_on 'Save and continue'
+    end
+
+    def fill_in_reason_for_prior_authority
+      fill_in 'Why is prior authority required?', with: 'Required because...'
+
+      # this emulates a drop of a file on the dropzone
+      # find(".dz-clickable").drop(file_fixture("basic_bulk_submission.csv"))
```

Comment:
I'm guessing this is commented out because it breaks for non-JSy tests? I've had a quick play, and it looks like `attach_file(Rails.root.join('spec/fixtures/files/test.png'))` works just fine - capybara knows to find the `file` input and attach the fixture to it.